docs/faq.md

```diff
@@ -167,3 +167,19 @@ echo "export PATH=\$PATH:/home/$(whoami)/.local/bin" >> ~/.bashrc
 - Copy "Rancher Desktop" shortcut from Desktop and paste in Startup folder.
 - Restart your machine.
 ```
+#### **Where does Rancher Desktop actually put the data volumes?**
+
+**A:** 
+
+**Windows:**
+Open Run menu (Press Windows + R) and open the path provided below, depending on the active container runtime.
+```
+dockerd(moby): \\wsl$\rancher-desktop-data\var\lib\docker\volumes
+containerd: \\wsl$\rancher-desktop-data\var\lib\nerdctl\dbb19c5e\volumes\<namespace>
```

Comment:
FWIW, [nerdctl docs](https://github.com/containerd/nerdctl/blob/b235a096085039ad4238ee5c70dbf6bfa11da76f/docs/dir.md#datarootaddrhash) explain where `dbb19c5e` comes from; I think that means it's going to be a constant for us.